### DISCUSSION TITLE
Windows Portable App option

### DISCUSSION THREAD

**Discussion by yahooshua** [debauchee/barrier#1408] [Ideas]
Would it be possible to make a Windows portable App?

I do a lot of client onsite work. I find myself bouncing back and forth between my laptop and the client's computer. Or manually copying text or files from my laptop. Standard IT person stuff.

I would like to just plug in a USB stickto the client computer, auto start Barrier and have it instantly connect to my laptop through the network. Then use the client's keyboard/mouse to access my laptop as needed and be able to copy text or installation files. 

Can't find this option for Synergy, 'Mouse without borders'

**Comment by shymega** [comment]
This is covered by #961, please do searches on the tracker. I'll lock this for now, but if users need this discussion, we can reopen this discussion.
